components/employees/EditForm.tsx

```diff
@@ -0,0 +1,244 @@
+    updateEmployee
+  );
+
+  const [formData, setFormData] = useState<EmployeesData>({
+    _id: employee._id,
+    firstName: employee.firstName,
+    lastName: employee.lastName,
+    gender: employee.gender,
+    emailAddress: employee.emailAddress,
+    physicalAddress: employee.physicalAddress,
+    phoneNumber: employee.phoneNumber,
+    emergencyPhoneNumber: employee.emergencyPhoneNumber,
+    bankName: employee.bankName,
+    bankAccountNumber: employee.bankAccountNumber,
+    accountName: employee.accountName,
+    nextOfKinFullName: employee.nextOfKinFullName,
+    nextOfKinPhoneNumber: employee.nextOfKinPhoneNumber,
+    nextOfKinRelationship: employee.nextOfKinRelationship,
+    employmentRole: employee.employmentRole,
+    employmentStartDate: employee.employmentStartDate,
+    dateOfBirth: employee.dateOfBirth,
+    educationalLevel: employee.educationalLevel,
+  });
+
+  const submit = async () => {
+    try {
+      const result = await trigger({
+        ...data,
+        _id: employee._id,
+        firstName: employee.firstName,
+        lastName: employee.lastName,
+        gender: employee.gender,
+        emailAddress: employee.emailAddress,
+        physicalAddress: employee.physicalAddress,
+        phoneNumber: employee.phoneNumber,
+        emergencyPhoneNumber: employee.emergencyPhoneNumber,
+        bankName: employee.bankName,
+        bankAccountNumber: employee.bankAccountNumber,
+        accountName: employee.accountName,
+        nextOfKinFullName: employee.nextOfKinFullName,
+        nextOfKinPhoneNumber: employee.nextOfKinPhoneNumber,
+        nextOfKinRelationship: employee.nextOfKinRelationship,
+        employmentRole: employee.employmentRole,
+        employmentStartDate: employee.employmentStartDate,
+        dateOfBirth: employee.dateOfBirth,
+        educationalLevel: employee.educationalLevel,
+      });
+      if (result) {
+        result
+        router.push('/employees')
+      }
+    } catch (e) {
+      console.log(e);
+      return;
+    }
+  };
+
+  return (
+    <View className="mb-24">
+      <View className="p-8 h-15">
+        <Text className="text-xl text-neutral-100">
+          Enter employee's details
+        </Text>
+      </View>
+      <ScrollView className="px-6 bg-neutral-800">
+        <EditInput
+          title="First name"
+          value={formData.firstName}
+          placeholder="Enter first name"
+          defaultValue={employee.firstName}
+          handleChangeText={(firstName: string) =>
+            setFormData({ ...formData, firstName })
+          }
+        />
+        <EditInput
+          title="Last name"
+          value={employee.lastName}
+          placeholder="Enter last name"
+          defaultValue={employee.lastName}
+          handleChangeText={(lastName: string) =>
+            setFormData({ ...formData, lastName })
+          }
+        />
+        <Radio
+          options={[
+            { label: "MALE", value: "MALE" },
```

Comment:
You can also use enum here too

Reply:
Okay, I did initially but I then hardcoded it. I'll revert it

Reply:
Done!